packages/smooth_app/lib/pages/product/common/product_list_page.dart

```diff
@@ -53,6 +53,19 @@ class _ProductListPageState extends State<ProductListPage>
     productList = widget.productList;
   }
 
+  //returns bool to handle WillPopScope
+  Future<bool> _handleUserBacktap() async {
+    if (_selectionMode) {
+      setState(() {
+        _selectionMode = false;
+        _selectedBarcodes.clear();
+      });
```

Comment:
You could add a comma, for refactoring 
```suggestion
      },);
```

Reply:
Sure, but I think it would make setState a bit less readable? Because ( ( ){ } ) gets kind of confusing when formatted.